roles/fips/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: 'Install dracut-fips'
+  action: yum name=dracut-fips state=installed
```

Comment:
We tend to space it out a bit with newlines.

Reply:
line breaks are added.